### ISSUE TITLE
Can't import module

### ISSUE THREAD

**Issue by itaydressler** [casesandberg/react-reactions#2]
I can't seem to be able to integrate this module. (I use type script). 

Keep getting this - 'Error: Can't resolve 'react-reactions' '.

Tried :
const FacebookSelector = require('react-reactions');
import * as Selectors from 'react-reactions';
import { FacebookSelector } from 'react-reactions';

None seem to work. I'll appreciate any help.

**Comment by praisegeek** [comment]
Same issue here.

**Comment by iainbryson** [comment]
Same here.  I'm using yarn and the import fails to find `react-reactions` despite it being in `node_modules`

**Comment by danfernand** [comment]
The problem is that the main file is pointing to lib/index but there is not lib folder. There is not transpile run command either.

**Comment by Groovej** [comment]
Do you have any suggestions on that?

**Comment by yingkeen** [comment]
Same issue here.

**Comment by khawerrind** [comment]
I am also having the same issue, Did any of you figured out the issue?

**Comment by anhmt90** [comment]
I also have the same issue. Please fix the bug.

**Comment by fuchsvomwalde** [comment]
Hey folks, here is a workaround to get the missing `lib` folder created after installing your project dependencies (important, so that it will also work in CI builds etc.):
just add the following script to your `package.json`:
`"postinstall": "yarn --cwd ./node_modules/react-reactions/"`

Hope this might help you until the bug will be finally fixed.

**Comment by amitneeman** [comment]
Any plans on fixing this one up? Ive seen there is a pull request ready to be pushed

**Comment by randomdipesh** [comment]
The lib folder was not getting pushed to github as it was added on .gitignore  . I faced the same problem as other commenters  and cloned the repo and installed its all dependencies and compiled the project  , removed lib from gitignore and tested and it worked .
For anyone facing this problem , install the fixed version of this module . 
You can install it using 
`npm i randomdipesh/react-reactions`

**Comment by raghav4** [comment]
@randomdipesh I'm getting undefined when I'm using it, can you please help?

**Comment by randomdipesh** [comment]
@raghav4  did you tried my repo? 
Install it using ,
`npm i randomdipesh/react-reactions`

**Comment by raghav4** [comment]
> @raghav4  did you tried my repo? 
> 
> Install it using ,
> 
> `npm i randomdipesh/react-reactions`

@randomdipesh  Yes I did and it worked but I was asking how should I use it? I imported the component and used it `<SlackCounter/>` but it showed nothing, what I'm asking is how do I show the emojis 

**Precisely** I want to use the `<GithubCounter/>` or `<SlackCounter/>` with the support of counter values, can you please suggest me how do I use it?

Here's what I'm Want
<img width="361" alt="Screenshot 2020-05-02 at 5 31 21 PM" src="https://user-images.githubusercontent.com/20716185/80863551-d7b46f00-8c9a-11ea-9618-78694fc6e662.png">

**Right now I'm getting this,** 

<img width="122" alt="Screenshot 2020-05-02 at 5 32 17 PM" src="https://user-images.githubusercontent.com/20716185/80863554-e0a54080-8c9a-11ea-8e42-3db473fe92a3.png">

**Comment by prathik-naidu** [comment]
@raghav4 Did you end up resolving this issue? Happening to me as well

**Comment by raghav4** [comment]
It's been so long @prathik-naidu, but what I remember is you need to pass the array of emoji's to make it work. Try doing that, let me know if that works.

**Comment by ARamss** [comment]
> @raghav4 did you tried my repo?
> Install it using ,
> `npm i randomdipesh/react-reactions`

Your repo is not installing on my project: 

```
npm install randomdipesh/react-reactions
npm ERR! code ENOENT
npm ERR! syscall spawn git
npm ERR! path git
npm ERR! errno -4058
npm ERR! enoent Error while executing:
npm ERR! enoent undefined ls-remote -h -t ssh://git@github.com/randomdipesh/react-reactions.git
npm ERR! enoent
npm ERR! enoent
npm ERR! enoent spawn git ENOENT
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\ramos\AppData\Roaming\npm-cache\_logs\2021-01-03T03_53_00_993Z-debug.log
```

**Comment by ARamss** [comment]
> @raghav4 did you tried my repo?
> Install it using ,
> `npm i randomdipesh/react-reactions`

Your repo is not installing on my project:

npm install randomdipesh/react-reactions
npm ERR! code ENOENT
npm ERR! syscall spawn git
npm ERR! path git
npm ERR! errno -4058
npm ERR! enoent Error while executing:
npm ERR! enoent undefined ls-remote -h -t ssh://git@github.com/randomdipesh/react-reactions.git
npm ERR! enoent
npm ERR! enoent
npm ERR! enoent spawn git ENOENT
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\ramos\AppData\Roaming\npm-cache\_logs\2021-01-03T03_53_00_993Z-debug.log

**Comment by ARamss** [comment]
@randomdipesh 

Your repo is not installing on my project:

npm install randomdipesh/react-reactions
npm ERR! code ENOENT
npm ERR! syscall spawn git
npm ERR! path git
npm ERR! errno -4058
npm ERR! enoent Error while executing:
npm ERR! enoent undefined ls-remote -h -t ssh://git@github.com/randomdipesh/react-reactions.git
npm ERR! enoent
npm ERR! enoent
npm ERR! enoent spawn git ENOENT
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! A complete log of this run can be found in:
npm ERR! C:\Users\ramos\AppData\Roaming\npm-cache_logs\2021-01-03T03_53_00_993Z-debug.log

**Comment by ARamss** [comment]
@randomdipesh

Your repo is not installing on my project:

```
npm install randomdipesh/react-reactions
npm ERR! code ENOENT
npm ERR! syscall spawn git
npm ERR! path git
npm ERR! errno -4058
npm ERR! enoent Error while executing:
npm ERR! enoent undefined ls-remote -h -t ssh://git@github.com/randomdipesh/react-reactions.git
npm ERR! enoent
npm ERR! enoent
npm ERR! enoent spawn git ENOENT
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! A complete log of this run can be found in:
npm ERR! C:\Users\ramos\AppData\Roaming\npm-cache_logs\2021-01-03T03_53_00_993Z-debug.log
```

**Comment by charkour** [comment]
@ARamss, I have [forked this repo](https://github.com/charkour/react-reactions/) and have [published a working version](https://www.npmjs.com/package/@charkour/react-reactions) of the library (with TypeScript support). 

```sh
npm i @charkour/react-reactions
```

**Comment by Miltonr87** [comment]
Amazing @charkour !

**Comment by xtrinch** [comment]
Thanks @charkour!

**Comment by ARamss** [comment]
thanks a lot @charkour
